src/release-notes/v2021.08.16-0.md

```diff
@@ -10,3 +10,6 @@ weight: 202108160
 - Added [Prometheus add-on](/docs/add-ons/prometheus) version 0.49.0-17.1.3.
 - Added [local-volume-provider](https://github.com/replicatedhq/local-volume-provider) plugin to Velero addon versions 1.5.1 through 1.6.2.
 - Added [KOTS add-on](/docs/add-ons/kotsadm) version 1.50.0
+
+### <span class="label label-orange">Bug Fixes</span>
+- Fixed case where docker preflights would docker is not enabled
```

Comment:
this is horribly worded
fix what? show?

Reply:
updated that to say `- Fixed case where docker preflights would run when docker is not enabled`